requirements-dev.txt

```diff
@@ -10,6 +10,7 @@ numpydoc
 openpyxl
 pandas
 pillow
+protobuf>=3.12.0,<4.0dev
```

Comment:
I've been seeing the protobuf change break a few things now.

Reply:
It'll probably take some time for libraries to work with version 4. Hopefully, once that has been achieved, we don't forget about this restriction.